src/rospack.cpp

```diff
@@ -1413,6 +1413,8 @@ Rosstackage::addStackage(const std::string& path)
   if((manifest_name_ == ROSSTACK_MANIFEST_NAME && stackage->isPackage()) ||
      (manifest_name_ == ROSPACK_MANIFEST_NAME && stackage->isStack()))
   {
+    if (stackage)
```

Comment:
Can you please remove the condition since `stackage` is always not null in this case.